src/www/system_general.php

```diff
@@ -229,6 +232,13 @@
         system_timezone_configure();
         filter_configure();
 
+        if ($timezone_changed) {
+            // force real radvd restart
+            $radvd = find_service_by_name('radvd');
+            killbypid($radvd['pidfile'], 'TERM', true);
+            services_radvd_configure();
```

Comment:
Maybe we should either fix the reload behaviour of radvd here or pass an argument to do it. HUP is a nice feature, but it never really works in edge cases?